app/src/main/java/agency/five/codebase/android/movieapp/ui/component/CrewItem.kt

```diff
@@ -0,0 +1,49 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+data class CrewItemViewState(
+    val name: String,
+    val job: String
+)
+
+@Composable
+fun CrewItem(
+    viewState: CrewItemViewState
```

Comment:
Every composable function should take modifier as parameter and pass it to a top level component!
Fix this in all your composable functions.